pytextnow/TNAPI.py

```diff
@@ -28,16 +29,16 @@
 class Client:
     def __init__(self, username: str = None, sid_cookie=None, csrf_cookie=None):
         # Load SIDS
-        self._user_sid = {}
+        self._user_cookies = {}
         self._good_parse = False
-        self._user_sid_file = join(dirname(realpath(__file__)), 'user_sids.json')
+        self._user_cookies_file = join(dirname(realpath(__file__)), 'user_sids.json')

```

Comment:
Filename needs to be changed to `user_cookies.json`

Reply:
thnx